gtest/src/manager.rs

```diff
@@ -348,7 +351,7 @@ impl JournalHandler for ExtManager {
         }
     }
     fn update_nonce(&mut self, program_id: ProgramId, nonce: u64) {
-        if let Some((Program::Core(prog), _)) = self.programs.get_mut(&program_id) {
+        if let Some((Program::Core(prog), _, _)) = self.actors.get_mut(&program_id) {
```

Comment:
Two wildcards may be condensed if we don't care about their count.

```suggestion
        if let Some((Program::Core(prog), ..)) = self.actors.get_mut(&program_id) {
```

https://rust-lang.github.io/rustfmt/?version=v2.0.0-rc.2&search=#condense_wildcard_suffixes